EventVisualisation/View/src/Initializer.cxx

```diff
@@ -57,6 +59,7 @@ void Initializer::setup(const Options options, EventManager::EDataSource default
   eventManager.setCdbPath(ocdbStorage);
 
   eventManager.registerDetector(new DataReaderTPC(), new DataInterpreterTPC(), EVisualisationGroup::TPC);
+  eventManager.registerDetector(new DataReaderITS(), new DataInterpreterITS(), EVisualisationGroup::ITS);
```

Comment:
We should probably avoid the unconditional registering of the detectors. Should not this better be done based on the <DET>.draw flags inside the o2eve_config file ?  (Or, based on the information stored inside the GRP object ?)